lib/plugins/custom-keymap.js

```diff
@@ -0,0 +1,84 @@
+import { Selection } from "prosemirror-state";
+import { keymap } from "prosemirror-keymap";
+import { baseKeymap, chainCommands } from "prosemirror-commands";
+import { splitListItem } from "prosemirror-schema-list";
+
+const paragraphParentIsType = (type, head) => {
```

Comment:
I'm finding the name of this method a bit confusing - looks like it doesn't actually have to operate on a paragraph?